week-12/homework/homework-1.html

```diff
@@ -0,0 +1,229 @@
+<html lang="en">
+
```

Comment:
@TaiChiLe I do not think your VS Auto format on save is working correctly as the there is a lot of empty line and does not look like the page is formatted correctly

Reply:
I think i put in these empty lines